assets/airports/ksjc.json

```diff
@@ -482,48 +482,74 @@
     		"origin": "KSJC",
     		"destination": "",
     		"category": "departure",
-    		"route": "KSJC.TECKY3.EBAYE",
+    		"route": "KSJC30R.TECKY3.EBAYE",
     		"altitude": "",
     		"speed": "",
     		"method": "random",
-    		"rate": null,
+    		"rate": 7,
```

Comment:
Do these rates make sense? The yearly total ('19-'20) of IFR flights was 234,701. Divided by 8760 and divided amongst the spawnPatterns give ~5, but I bumped it up to 7.